plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties

```diff
@@ -5,3 +5,4 @@ appengine.blacklist.marker=App Engine Configuration Problem
 web.xml=Web XML Validator
 servlet.marker=Servlet Marker
 maven.plugin.marker=Obsolete Maven App Engine Plugin
+undefined.servlet.marker=Undefined Servlet Class
```

Comment:
If possible, we should parameterize this with {0} to include the name of the undefined class.